src/plugins/source-plugin.js

```diff
@@ -495,7 +495,11 @@ export default (options) =>
     const getAttribute = (tag, attribute, attributes, resourcePath) => {
       return attributeList.find(
         (element) =>
-          element.tag.toLowerCase() === tag.toLowerCase() &&
+          // eslint-disable-next-line no-undefined
+          (element.tag === undefined ||
+            // eslint-disable-next-line no-undefined
+            (element.tag !== undefined &&
```

Comment:
Use `type of value === "undefined"` to avoid disable the rule

Reply:
Done :+1: